lms/djangoapps/instructor/views/api.py

```diff
@@ -917,6 +922,7 @@ def list_course_role_members(request, course_id):
                 "email": "staff1@example.org",
                 "first_name": "Joe",
                 "last_name": "Shmoe",
+                "group_name": "Verified"
```

Comment:
Nit: the style in Python is to have the (extra) trailing comma. Same note about line 947 below.

One reason people like it is that if you add things to the list, you don't have to edit the previous line, and therefore the git history for the previous line is not impacted.